package.json

```diff
@@ -28,10 +28,13 @@
   },
   "homepage": "https://github.com/babel/broccoli-babel-transpiler",
   "dependencies": {
-    "babel-core": "^5.0.0",
-    "broccoli-persistent-filter": "^1.0.1",
+    "babel-core": "^6.0.0",
+    "babel-plugin-transform-es2015-modules-amd": "^6.0.0",
+    "babel-preset-es2015": "^6.0.0",
```

Comment:
The two deps above may live better in dev dependencies, but I don't know how many assumptions you're making about the usage of this tool.